tests/test_repository.py

```diff
@@ -21,7 +21,17 @@
 from twine import utils
 
 
+@pytest.fixture()
+def repo():
```

Comment:
Assuming we don't parameterize this, I think a more specific name would make it easier to understand what's going on in the tests (though it requires a global rename on `repo`).

```suggestion
def default_repo():
```

Reply:
I will make this change, assuming you are fine with the global renaming of `repo` to `default_repo` across the test functions using it.

Reply:
Fixed in latest commit